iothub/device/src/Authentication/IConnectionCredentials.cs

```diff
@@ -99,6 +99,6 @@ internal interface IConnectionCredentials
         /// <summary>
         /// Gets the SAS token credential required for authenticating the client with IoT hub service.
         /// </summary>
-        public Task<string> GetPasswordAsync();
```

Comment:
How did this even compile before?

Reply:
This was defined as a public method on the interface, however, it makes more sense to make this internal. This method is of relevance only to our SDK.